pyseir/inference/model_fitter.py

```diff
@@ -177,13 +224,9 @@ def __init__(
         self.dof_cases = None
         self.dof_hosp = None
 
-    @property
-    def state_fips(self):
-        return self.fips[:2]
-
     @property
     def display_name(self):
```

Comment:
fwiw i think this would be great function to have on say `Region` or `RegionalCombinedData`

Reply:
Done.